examples/todos-flow/src/components/Link.js

```diff
@@ -1,10 +1,10 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
 
 export type Props = {
   active: boolean,
-  children?: React$Element<any>,
+  children?: React.Node,
```

Comment:
`React$Node` avoids the wildcard import.

Reply:
I can also import `import React, { type Node } from 'react';`